MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include include *.h
```

Comment:
I could've sworn this used to not work, unless you also specified it in package_data. Nice!